opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -150,18 +182,20 @@ def __init__(
         description: str,
         unit: str,
         value_type: Type[metrics_api.ValueT],
+        meter: "Meter",
         label_keys: Sequence[str] = (),
         enabled: bool = True,
-        monotonic: bool = True,
+        alternate: bool = True,
     ):
         super().__init__(
             name,
             description,
             unit,
             value_type,
+            meter,
             label_keys=label_keys,
             enabled=enabled,
-            monotonic=monotonic,
+            alternate=alternate,
         )
 
     def add(self, label_set: LabelSet, value: metrics_api.ValueT) -> None:
```

Comment:
this is an aside, but can we flip the label_set and value argument ordering?

I feel like there's a common scenario where one would want to measure a metric with NO labels whatsoever (e.g. memory).

So I feel like we should support a label_set less api:

```
request_counter = Counter(...)
request_counter.add(1.0)
# maybe I want to add labels too
request_counter.add(1.0, {"caller": "upstream-a"})
```

Reply:
That's true. I think in the future if we support a label_set less api (extremely likely, as it will probably default to the empty labelset), it makes sense for the label_set field to be optional.